mlir/lib/Dialect/EmitC/IR/EmitC.cpp

```diff
@@ -929,22 +930,44 @@ LogicalResult emitc::VerbatimOp::verify() {
   return success();
 }
 
+static ParseResult parseVariadicFmtArgs(AsmParser &p,
```

Comment:
```suggestion
static ParseResult parseVariadicTypeFmtArgs(AsmParser &p,
```

Reply:
This is really specific to having a variadic number of types, so this should be made explicit in the parser's name.